threewisemonkeys-as_robo_interact_s2_159.py

```
@@ -0,0 +1,117 @@
+    model_start = time.time()
+
+    from segment_anything import sam_model_registry, SamAutomaticMaskGenerator
+    model_type = "vit_b"
+    
+    # Initialize the SAM model using the registry
+    sam = sam_model_registry[model_type](checkpoint=sam_checkpoint)
+    sam.to(device=device)
+    logger.info(f"Model loaded in {time.time() - model_start:.2f} seconds")
+    
+    # Initialize mask generator
+    mask_start = time.time()
```

Ruff (0.13.1)

Local variable `mask_start` is assigned to but never used

Remove assignment to unused variable `mask_start`

(F841)